service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/TimeToLiveTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.nhs.adaptors.gp2gp.ehr;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import uk.nhs.adaptors.gp2gp.common.mongo.MongoClientConfiguration;
+import uk.nhs.adaptors.gp2gp.common.mongo.ttl.MongoTtlCreator;
+import uk.nhs.adaptors.gp2gp.repositories.EhrExtractStatusRepository;
+import uk.nhs.adaptors.gp2gp.testcontainers.ActiveMQExtension;
+import uk.nhs.adaptors.gp2gp.testcontainers.MongoDBExtension;
+
+import org.assertj.core.api.Assertions;
+import org.awaitility.Durations;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.index.IndexInfo;
+import org.springframework.data.mongodb.core.index.IndexOperations;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith({ SpringExtension.class, MongoDBExtension.class, ActiveMQExtension.class})
+@SpringBootTest
+@DirtiesContext
+public class TimeToLiveTest {
+    private static final int MAX_AWAIT_IN_SECONDS = 90;
```

Comment:
src/main/resources/application.yml defines the default TTL for the app. The story does specify it, but it should be 7 days.

Reply:
The test fails when running below 40 seconds, leaving at 90 at discussed.